src/System.CommandLine/Command.cs

```diff
@@ -10,40 +10,83 @@
 
 namespace System.CommandLine
 {
+    /// <summary>
+    /// Represents a specific action that the application performs.
+    /// </summary>
+    /// <remarks>
+    /// Use the Command object for actions that correspond to a specific string (the command name). See
+    /// <see cref="RootCommand"/> for simple applications that only have one action. For example, <c>dotnet run</>
+    /// uses <c>run</c> as the command.
+    /// </remarks>
     public class Command : 
         NamedSymbol, 
         ICommand, 
         IEnumerable<Symbol>
     {
         private readonly SymbolSet _globalOptions = new SymbolSet();
 
+        /// <summary>
+        /// Initializes a new instance of the Command class.
+        /// </summary>
+        /// <param name="name">The name of the command.</param>
+        /// <param name="description">The description of the command, shown in help.</param>
         public Command(string name, string? description = null) : base(name, description)
         {
         }
 
+        /// <summary>
+        /// Represents all the arguments associated with the command.
+        /// </summary>
         public IEnumerable<Argument> Arguments => 
             Children.OfType<Argument>();
 
+        /// <summary>
+        /// Represents all the options associated with the command, including global options.
+        /// </summary>
         public IEnumerable<Option> Options =>
             Children.OfType<Option>()
                     .Concat(Parents
                             .OfType<Command>()
                             .SelectMany(c => c.GlobalOptions));
-
+        /// <summary>
+        /// Represents all the global options associated with the command
+        /// </summary>
         public IEnumerable<Option> GlobalOptions => _globalOptions.OfType<Option>();
 
+        /// <summary>
+        /// Adds an <see cref="Argument"/> to the command.
+        /// </summary>
+        /// <param name="argument">The argument to add to the command.</param>
         public void AddArgument(Argument argument) => AddArgumentInner(argument);
 
+        /// <summary>
+        /// Adds a subcommand to the command. Commands can be nested to an arbitrary depth.
+        /// </summary>
+        /// <param name="command">The subcommand to add to the command.</param>
         public void AddCommand(Command command) => AddSymbol(command);
 
+        /// <summary>
+        /// Adds an <see cref="Option"/> to the command.
+        /// </summary>
+        /// <param name="option">The option to add to the command.</param>
         public void AddOption(Option option) => AddSymbol(option);
 
+        /// <summary>
+        /// Adds a global <see cref="Option"/> to the command. Global options are applied to all commands.
```

Comment:
Global options are only applied to subcommands, recursively, but not to parent commands.

Reply:
I moved this to the remarks too, and expanded the text to be clear that it applies to the current command, subcommands but not parent commands. See what you think.